### PR TITLE
doc: add homebrew release badge

### LINKED ISSUE #147: Homebrew Support
It would be easier to install and upgrade the Himalaya cli if there was a formula in homebrew.

### PR DESCRIPTION
**Pull request by chenrui333**
fixes #147

### CODE REVIEW

**Comment by chenrui333** [comment]
@soywod I dont know how to update wiki so I filed a PR of adding the installation instruction in here.

**Comment by chenrui333** [comment]
Any update on this thread?

File: README.md [hunk 1]
```diff
@@ -29,6 +33,14 @@ curl -sSL https://raw.githubusercontent.com/soywod/himalaya/master/install.sh |
 curl -sSL https://raw.githubusercontent.com/soywod/himalaya/master/install.sh | PREFIX=~/.local sh
 ```
 
+### Install via brew
```
Comment (soywod) [inline README.md:36]: Instructions add to the wiki at https://github.com/soywod/himalaya/wiki/Installation:using-brew. Could you remove this from the PR?

**Review by soywod — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (chenrui333) [reply README.md:36]: Sounds good, actually this is the first project that I saw using wiki so seriously 😄

Comment (chenrui333) [reply README.md]: Just did the change. 👍

**Comment by soywod** [comment]
Thank you :tada: sorry for the delay, I was quite busy these days.

**Comment by chenrui333** [comment]
totally understand and thanks for the merge 🎉

**Pull request merged**